db/seeds.rb

```diff
@@ -10,3 +10,4 @@
 # the relevant ENV vars after it has run at least once.
 RolesConversion.new.ncr_budget_approvers
 RolesConversion.new.gsa18f_approvers
+RolesConversion.new.gateway_roles
```

Comment:
I assumed it should be in seeds because it's now an important role in the app, and so dev instances should have it. But we also do a `Role.find_or_create_by(name: 'gateway_admin')` in the `User` factory (for the `:gateway_admin` trait).